pyproject.toml

```diff
@@ -50,6 +50,7 @@ version = {attr = "weave.__version__"}
 dependencies = {file = ["requirements.txt"]}
 
 [tool.setuptools.dynamic.optional-dependencies]
+examples = {file = ["requirements.ecosystem.txt"]}
```

Comment:
A bit liberal for now, we can refine this later with a better scoped list of dependencies.